mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -1119,6 +1125,120 @@ static std::tuple<short, short, int> configRandomTestData() {
   return std::make_tuple(min, max, seed);
 }
 
+static void generateTensorInitValues(
+    OpBuilder &builder, Block *block, mlir::FuncOp &mcpuMemset5DFuncOp,
+    mlir::Value &filterMemsetMinValue, mlir::Value &filterMemsetMaxValue,
+    mlir::Value &inputMemsetMinValue, mlir::Value &inputMemsetMaxValue,
+    mlir::Value &outputMemsetMinValue, mlir::Value &outputMemsetMaxValue,
+    mlir::ConstantOp &seedConstantIntOp) {
+  auto int16Type = builder.getIntegerType(16);
+  auto int32Type = builder.getIntegerType(32);
+  unsigned short zero = 0, one = 1;
+  short min, max;
+  int seed = 1;
+  std::tie(min, max, seed) = configRandomTestData();
+
+  mlir::ConstantOp oneConstantIntOp;
```

Comment:
we could just set `minConstantIntOp` and `maxConsnatIntOp` be `1` and reduce the logic a little bit, by eliminating `oneConstantIntOp`.